docker/Dockerfile

```diff
@@ -0,0 +1,25 @@
+FROM ruby:2.5-slim
+
+#install packages
+#fixes dpkg man page softlink error while installing postgresql-client [source: https://github.com/debuerreotype/debuerreotype/issues/10#issuecomment-438342078]
+RUN mkdir -p /usr/share/man/man1 && mkdir -p /usr/share/man/man7
+RUN apt-get update && apt-get install curl -y
```

Comment:
Each time apt-get is run you should update and then remove the APT cache files at the end of the command.

Reply:
Why?

Reply:
So that the cache files don't make it into the docker image

Reply:
Isn't this just building a container for local use though? Not anything that's going to be published?

Reply:
Sure, but what I've read indicates it as a best practice in general